### ISSUE TITLE
Time to leave 2015 behind... 

### ISSUE THREAD

**Issue by radinamatic** [learningequality/ka-lite#5055]
## Summary

Footer still displays year 2015:

![year](https://cloud.githubusercontent.com/assets/1457929/14083097/2a8bd3f8-f514-11e5-8ca4-1611b963bf92.png)
 
I have PR practically ready, changed the `_footer.html`, but what about the line 297 in `kalite/settings/base.py`, does that need changing too? It's Khan Academy, not KA Lite:

```
 "footer_text": ugettext_lazy(u"Videos © 2015 Khan Academy (Creative Commons) // Exercises © 2015 Khan Academy"),
```

If somebody confirms this (@aronasorman, @jamalex), I can push the PR.

**Comment by aronasorman** [comment]
@radinamatic too late, we had the string freeze (aka uploaded the po files) 2 weeks ago. No more string changes at this point.

**Comment by radinamatic** [comment]
That "string" is a year number, how important is it to be translated? :stuck_out_tongue: 
@aronasorman

**Comment by aronasorman** [comment]
I see that string translated sometimes, so I can't assume that it's unimportant for translators. I'm holding steadfast on this one.

**Comment by radinamatic** [comment]
Ok, ok... 
It's just a pity to have the last year there... :anguished:

**Comment by aronasorman** [comment]
Yeah, alas...

feel free to open a PR targeting develop and 0.16.x, so that it can be merged in once 0.16.0 has been released!

**Comment by benjaoming** [comment]
@aronasorman this was a straight-forward fix, I don't think the string freeze policy was warranted.. will release next time around.

**Comment by benjaoming** [comment]
Ah okay not all of it, but most of it, I'll get to the channel footer text, too when I figure out string formatting for `ugettext_lazy`.

**Comment by radinamatic** [comment]
Issue is still "officially" assigned to me, but not sure if your PR fixed it and can be closed, or if it's still WIP...? @benjaoming

**Comment by benjaoming** [comment]
It was only partially fixed as you'll see in the footer of 0.16.6 :)

**Comment by benjaoming** [comment]
Fixed in #5307
